adobe/aio-theme
# Attribution and Edition features

## Description

This PR adds both Edition and Attribution components to satisfy the documentation requirements for Commerce DevDocs, as specified in these two Jira tickets:

- https://jira.corp.adobe.com/browse/COMDOX-76
- https://jira.corp.adobe.com/browse/COMDOX-77

## Motivation and Context

These enhancements are necessary to provide features for editions and partner attributes within Commerce devdocs.

## How Has This Been Tested?

Local testing with frontmatter that matches the frontmatter for these features in devdocs pages. 

## Screenshots (if appropriate):

![image](https://user-images.githubusercontent.com/1828494/167424450-dccd2ad2-11c5-4603-bb28-eafd2ef32a9e.png)

## How to Test

![image](https://user-images.githubusercontent.com/1828494/167450663-76c47f25-5416-498a-895a-f6eb93c8cc66.png)

1. Pull down the branch locally.
2. Open a terminal in the root directory.
3. Run `yarn` or `yarn install` (same thing) to download and install the packages.
4. Run `yarn dev` to build the theme and the example site.
5. If the build succeeds, go here: http://localhost:8000/support/FAQ/
6. You should see something like the screen shot above.
7. Now navigate to the FAQ page from the project root: `example/src/pages/support/FAQ/index.md`
8. Change any of the following frontmatter: 
     `contributor_name: Kalpesh Mehta from Corra`
     `contributor_link: https://partners.magento.com/portal/details/partner/id/70/`
     `edition: ee`
9. With each change, you need to save the file, stop your local server, and rebuild the project by runing `yarn dev` again to see the FAQ page changes.

The reason you need to stop the server and rebuild between each change is because the GraphQL query that retrieves the frontmatter data only runs once per build and stored for the application to use. So if you change the frontmatter, you need to rerun the query—rebuild the project.


## Types of changes




## Checklist:
